.github/workflows/test.yml

```diff
@@ -1,7 +1,5 @@
 name: Test
```

Comment:
@putmantime The test action. Looking at bottom of the pull request, I see that the tests passed. Sweet!